src/Tomcat/TomcatServer.ts

```diff
@@ -15,10 +19,11 @@ export class TomcatServer {
     private _isDebugging: boolean = false;
 
     constructor(name: string, tomcatPath: string, extensionPath: string) {
-        this._name = name.trim();
-        this._tomcatPath = tomcatPath.trim();
+        this._name = name;
+        this.label = name;
+        this._tomcatPath = tomcatPath;
```

Comment:
use TypeScript Constructor Assignment to reduce a lot of codes. https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/